css/css-masking/clip-path-svg-content/WEB_FEATURES.yml

```diff
@@ -0,0 +1,6 @@
+features:
+- name: clip-path-boxes
+  files:
+  - clip-path-shape-circle-003.svg
+  - clip-path-shape-circle-004.svg
+  - clip-path-shape-circle-005.svg
```

Comment:
Add [clip-path-inset-stroke-001.svg](https://wpt.fyi/results/css/css-masking/clip-path-svg-content/clip-path-inset-stroke-001.svg?label=experimental&label=master&aligned) and [clip-path-inset-stroke-002.svg](https://wpt.fyi/results/css/css-masking/clip-path-svg-content/clip-path-inset-stroke-002.svg?label=experimental&label=master&aligned)

Reply:
Thank you, added now!